scripts/git-heat.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Build a commit frequency list.
+
+function git-heat() {
```

Comment:
Why write this as a function (that must be `source`d before it can be run) instead of making the `scripts/git-heat.sh` the script itself?

Reply:
With the function wrapper, I do have to `source`, but then I get tab completion in my shell. Without it, tab completion isn't working. Any idea why?

I compared with `scripts/codechecks/hqDefine.sh` which does tab complete, but I don't see what's different about it.

Reply:
Offlined, making updates in https://github.com/dimagi/commcare-hq/pull/28720